realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -73,6 +73,7 @@ public RealmQuery(Realm realm, Class<E> clazz) {
         this.realm = realm;
         this.clazz = clazz;
         this.table = realm.getTable(clazz);
+        view = null;
```

Comment:
There are two reasons to assign null here.

One is to clarify `view` is null as you mentioned.
Another is that it is mandatory in order to make `view` field final.